CMakeLists.txt

```diff
@@ -12,11 +15,18 @@ if(NOT PYTHON_EXECUTABLE)
 endif()
 
 message(STATUS "Running gen_cmake_skeleton.py")
+option(BUILD_SHARED_LIBS "Build shared Kaldi libraries." OFF)

```

Comment:
Your previous PR got intermixed with this one. Perhaps you should merge Kaldi master into your branch (depends on your Git-Fu level), but a simple merge will take care of this.